services/src/main/java/org/exoplatform/task/rest/TaskRestService.java

```diff
@@ -628,14 +629,15 @@ public Response getTaskComments(@ApiParam(value = "Task id", required = true) @P
   }
 
   @POST
-  @Path("comments/{id}")
+  @Path("comments/{id}/{lang}")
   @RolesAllowed("users")
   @ApiOperation(value = "Adds comment to a specific task by id", httpMethod = "POST", response = Response.class, notes = "This Adds comment to a specific task by id")
   @ApiResponses(value = { @ApiResponse(code = 200, message = "Request fulfilled"),
       @ApiResponse(code = 400, message = "Invalid query input"), @ApiResponse(code = 403, message = "Unauthorized operation"),
       @ApiResponse(code = 404, message = "Resource not found") })
   public Response addTaskComment(@ApiParam(value = "Comment text", required = true) String commentText,
-                                 @ApiParam(value = "Task id", required = true) @PathParam("id") long id) {
+                                 @ApiParam(value = "Task id", required = true) @PathParam("id") long id,
+                                 @ApiParam(value = "Lang", required = true) @PathParam("lang") String lang){
```

Comment:
here you have the current user so you can have his language. No need to put it in parameter